exporter/lambda/cross_region_import_replication.py

```diff
@@ -71,7 +71,7 @@ def _lambda_handler():
             )
         elif "No updates are to be performed." in message:
             print("No updates are to be performed.")
-        elif "UPDATE_IN_PROGRESS" in message:
+        elif "IN_PROGRESS" in message:
```

Comment:
I'd prefer if you use an `or` statement and check both statuses explicitly. This would avoid catching by accident statuses like ROLLBACK_IN_PROGRESS